src/components/Dropdown/components/Label/index.test.tsx

```diff
@@ -70,13 +78,29 @@ describe('Label', () => {
         name: 'with danger and secondaryLabel',
         item: { id: '1', label: 'Some Label', secondaryLabel: 'Secondary Label', danger: true },
       },
+      {
+        name: 'with tag',
+        item: { id: '1', label: 'Some Label', secondaryLabel: 'Secondary Label', tag: <Tag>Tag</Tag> },
+
+      },
     ]
 
     it.each(testCases)('render $name', ({ item }) => {
       const { baseElement } = renderLabel({ item, layout: ItemLayout.Vertical })
       expect(baseElement).toMatchSnapshot()
     })
   })
+
+  describe('tag', () => {
+    it('throws an error if tag is not a Tag component', () => {
+      const renderFn = (): RenderResult => renderLabel({
+        item: { id: '1', label: 'Some Label', tag: <div id="definitely-not-a-Tag" /> },
+        layout: ItemLayout.Horizontal,
+      })
+
+      expect(renderFn).toThrow('`item.tag` must be a Tag component')
```

Comment:
the item id is still missing, can we set it in the message?

Reply:
done